governance/policies/archiving.md

```diff
@@ -0,0 +1,102 @@
+# opam Repository Archiving Policy
+
+*NOTE*: This policy is not static and will be updated as the process matures.
+
+## Terminology
+
+- The primary opam repository, (referred to here as the "primary repo") is located at [ocaml/opam-repository](https://github.com/ocaml/opam-repository). The primary repo is curated to ensure that compatible packages are co-installable on as many supported platforms as possible, and it is the default package repository.
+- "Primary repo criteria" refers to the criteria used to decide whether a version of a package is suitable for continued inclusion in the primary repo.
+- The archive opam repository, (referred to here as the "archive repo")" is located at [ocaml/opam-repository-archive](https://github.com/ocaml/opam-repository-archive) and records packages that were once in the primary repo, but no longer meet the primary repo criteria.
+- "Supported platforms" are those listed under [Tier 1 and Tier 2 by the OCaml compiler](https://github.com/ocaml/ocaml?tab=readme-ov-file#overview)
+- A package's "maximum compiler version" is the upper bound of the OCaml compiler versions supported by a package. Support is derived either based on explicit version bounds set on the `ocaml` dependency in a package's dependency cone, or implicitly based on failed installs detected through our CI and health check processes.
+- The "compiler cutoff threshold" is an OCaml compiler versions stipulated by the opam repository maintainers. It sets a lower bound on the compiler versions supported by the primary repo. 
+- A package is said to "meet the compiler cutoff threshold" iff its maximum compiler version is greater then or equal to the compiler cutoff threshold.
+- The "maintenance intent" of a package refers to the explicitly declared intent of a maintainer to support versions of a package, recorded in the packages `opam` file metadata.
+
+## Policy
+
+### Compiler cutoff threshold
+
+The current compiler cutoff threshold is `4.08`.
+
+This threshold is subject to change by the opam repo maintainers based on the oldest compiler available on maintained distributions.
```

Comment:
It's worth cross referencing to what the maintained distributions are here

Reply:
Good point! What is the best reference for this? https://ocaml.org/tools/compiler-release-cycle or are you thinking of referencing the compiler versions in linux distros with active LTS support?

Reply:
So we could have a 4.05 threshold now and in May make another PR to move all packages < 4.08

Reply:
Oh no, I had read it wrong, it is 4.08 on 20.04 which goes EOL in May, then no change in the plan is really needed since we were supporting still 4.08 in the first cut